libs/designer-ui/src/lib/panel/__test__/searchResult.spec.tsx

```diff
@@ -0,0 +1,130 @@
+import type { OperationSearchCardProps, OperationSearchGroupProps } from '../recommendationpanel';
+import type { SearchResultsGridProps } from '../recommendationpanel/searchResult';
+import { SearchResultsGrid } from '../recommendationpanel/searchResult';
+import type { IListProps } from '@fluentui/react/lib/List';
+import type { DiscoveryOperation, DiscoveryResultTypes } from '@microsoft/utils-logic-apps';
+import type { ReactElement } from 'react';
+import React from 'react';
+import * as ReactShallowRenderer from 'react-test-renderer/shallow';
+
+describe('ui/workflowparameters/workflowparameter', () => {
+  let renderer: ReactShallowRenderer.ShallowRenderer;
+  let minimal: SearchResultsGridProps;
+
+  beforeEach(() => {
+    minimal = {
+      displayRuntimeInfo: false,
+      isLoadingMore: false,
+      isLoadingSearch: false,
+      onConnectorClick: jest.fn(),
+      onOperationClick: jest.fn(),
+      operationSearchResults: [],
+      searchTerm: '',
+    };
+    renderer = ReactShallowRenderer.createRenderer();
+  });
+
+  afterEach(() => {
+    renderer.unmount();
+  });
+
+  it('should group aliased connectors separately', () => {
+    renderer.render(<SearchResultsGrid {...minimal} groupByConnector={true} operationSearchResults={[operation, aliasedOperation]} />);
+
+    const grid = renderer.getRenderOutput();
+    const list = findByName<IListProps<string>>('List', grid);
+    expectValidReactElement(list);
+    expect(list.props.items).toHaveLength(2);
+    expect(list.props.items).toContain('api_id');
+    expect(list.props.items).toContain('displayApi_id');
+
+    const cell = list.props.onRenderCell!('displayApi_id');
```

Comment:
```suggestion
    const cell = list.props.onRenderCell?.('displayApi_id');
```

Please fix linter errors as well, for non null assertion I suggestion option unwrapping like the suggestion above

Reply:
Ok thx, this didn't show up when running locally npm run lint, I'll update.